src/Cake.Common/Tools/GitVersion/GitVersionAliases.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+
+namespace Cake.Common.Tools.GitVersion
+{
+    /// <summary>
+    /// Contains funcionality related to GitVersion.
+    /// </summary>
+    [CakeAliasCategory("GitVersion")]
```

Comment:
@RichiCoder1  When you come back around to this, would it be possible to change to to be GitTools, and then add an additional alias category to the GitVersion call below?  Similar to what is done here in the ReSharper tools? 

https://github.com/cake-build/cake/blob/develop/src/Cake.Common/Tools/DupFinder/DupFinderAliases.cs

Eventually, I would like to add aliases for all the GitTools, including GitReleaseManager, GitReleaseNotes, as well as GitVersion.  Assuming that @patriksvensson and @devlead  is ok with that :smile_cat: